### ISSUE TITLE
Cannot Install Jupyter

### ISSUE THREAD

**Issue by KyleSeebach** [JuliaLang/IJulia.jl#893]
When I am trying to install Jupyter, I get the error message in the text document. How do I fix this?
[installingJupyter.txt](https://github.com/JuliaLang/IJulia.jl/files/4063240/installingJupyter.txt)

**Comment by stevengj** [comment]
You don't seem to have permissions set correctly on you `C:\Users\Kyle Seebach\AppData\Local\Temp` directory?  Can you fix the permissions?

**Comment by KyleSeebach** [comment]
I am trying to fix the permissions now. I am following the instructions
from this website,
https://customer.real.com/hc/en-us/articles/204039863-Permission-to-write-to-TEMP-folder-and-Windows-system-folders.
Will this work?

**Comment by KyleSeebach** [comment]
I followed the instructions on that website, but the permissions did not change? What do you suggest I do?

**Comment by stevengj** [comment]
Did you look specifically at `AppData\Local\Temp` and `AppData\Local`?

**Comment by KyleSeebach** [comment]
Yes, but the permissions change back to read-only right after I apply the
changes.

**Comment by rifkiaditya** [comment]
I have problem same with you, how to solve it? Thx.

This is my problem when I wanna install jupyter by conda :

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
ERROR conda.core.link:_execute(700): An error occurred while installing package 'defaults::qt-5.9.7-vc14h73c81de_0'.
Rolling back transaction: done

LinkError: post-link script failed for package defaults::qt-5.9.7-vc14h73c81de_0
location of failed script: C:\ProgramData\Anaconda3\envs\eolearn\Scripts\.qt-post-link.bat
==> script messages <==
<None>
==> script output <==
stdout:         1 file(s) copied.

stderr: 'chcp' is not recognized as an internal or external command,
operable program or batch file.
'chcp' is not recognized as an internal or external command,
operable program or batch file.
'chcp' is not recognized as an internal or external command,
operable program or batch file.

return code: 1

()
 
anyone can help me?

**Comment by nilshg** [comment]
Can be closed.
